.changeset/green-actors-argue.md

```diff
@@ -0,0 +1,20 @@
+---
+'@backstage/cli': minor
+---
+
+**BREAKING**: Removed the following deprecated package commands:
+
+- `app:build` - Use `package build` instead
+- `app:serve` - Use `package build` instead
+- `backend:build` - Use `package build` instead
+- `backend:bundle` - Use `package build` instead
```

Comment:
is this in build? 🤔

Reply:
Yep, bundle == build at this point, just depends on the role